src/index.ts

```diff
@@ -1,67 +1,62 @@
-export {};
-
-const moment = require('moment');
-
+const moment = require("moment");
```

Comment:
@alpertl Could also use modern imports here and add moment types to package.json.